xapian-letor/tests/api_letor.cc

```diff
@@ -225,21 +225,7 @@ DEFINE_TESTCASE(createfeaturevectorthree, generated)
 DEFINE_TESTCASE(checkemptyfeaturelist, generated)
 {
     vector<Xapian::Feature*> f;
-    // pass empty feature list.
-    Xapian::FeatureList fl(f);
-    Xapian::Database db = get_database("db_index_two_documents",
-				       db_index_two_documents);
-
-    Xapian::Enquire enquire(db);
-    enquire.set_query(Xapian::Query("tigers"));
-    Xapian::MSet mset;
-    mset = enquire.get_mset(0, 10);
-
-    TEST(!mset.empty());
-
-    TEST_EXCEPTION(Xapian::InvalidArgumentError,
-		   fl.create_feature_vectors(mset, Xapian::Query("tigers"),
-					     db));
+    TEST_EXCEPTION(Xapian::InvalidArgumentError, Xapian::FeatureList fl(f));
```

Comment:
This no longer uses any database, let alone a `generated` one, so the testcase condition is now wrong - we use `!backend` for such tests (so they're run once in their own pass, rather than pointlessly run once per backend the test harness is testing over).

Reply:
I'd also suggest not including `check` (or `test`, etc) in the testcase name - it doesn't really add useful information and tends to result in rather long test case names.

Reply:
fixed in a fixup commit.